redesign/tests/app/AttributionDialogue.tests.js

```diff
@@ -2,17 +2,21 @@
 
 QUnit.module( 'AttributionDialogue' );
 
-var AttributionDialogue = require( '../../js/app/AttributionDialogue' ),
-	Messages = require( '../../js/app/Messages' );
+var $ = require( 'jquery' ),
+	AttributionDialogue = require( '../../js/app/AttributionDialogue' ),
+	Messages = require( '../../js/app/Messages' ),
+	Author = require( '../../js/app/Author' ),
+	Asset = require( '../../js/app/Asset' );
 
 QUnit.test( 'should have 4 steps by default', function( assert ) {
-	var dialogue = new AttributionDialogue();
+	var dialogue = new AttributionDialogue( new Asset( '', '', null, null, [] ) );
```

Comment:
When it gets this verbose, it probably makes sense to create a newDefaultAttributionDialogue function rather than repeating it so many times.